recipes/busco/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin/
+cp BUSCO_v1.2.py $PREFIX/bin
+ln -s $PREFIX/bin/BUSCO_v1.2.py $PREFIX/bin/BUSCO_v1.1b1.py
```

Comment:
I must say that I prefer to have an executable called ``busco``. Do you think this would be feasible? At least, I definitly don't want busco 1.2 to install a file indicating version 1.1b1 into bin.